css/selectors/state.json

```diff
@@ -8,8 +8,8 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/semantics-other.html#selector-custom",
           "support": {
             "chrome": {
-              "version_added": false,
-              "notes": "From version 90 Chromium supports custom state selection using a dashed-ident (such as <code>:--foo</code>) instead of <code>:state()</code>."
+              "version_added": "125",
+              "notes": "Before version 125, custom state selection was supported using a dashed-ident (such as <code>:--foo</code>) instead of <code>:state()</code>."
```

Comment:
So not sure if this is "sufficient" as a note, and even if this is sufficient, it slightly implies that the dashed ident form is no longer allowed. I'm not messing with it because the BCD team know better than me.

Reply:
Thanks @hamishwillee, I think you are proposing the following? 
Do you want to make that change, @rachelandrew?

```
"chrome": [
  {
    "version_added": "125"
  },
  {
    "version_added": "90",
    "notes": "Supported using a dashed-ident (such as <code>:--foo</code>) instead of <code>:state()</code>."
  }
]
```

Reply:
If the deprecated syntax will ever get removed, we could then have:

```
"chrome": [
  {
    "version_added": "125"
  },
  {
    "version_added": "90",
    "version_removed: "200",
    "notes": "Supported using a dashed-ident (such as <code>:--foo</code>) instead of <code>:state()</code>."
  }
]
```

Reply:
@Elchi3 Yes, that second entry captures the "intent" - a separate behaviour that co-exists. The plan is to remove it eventually.

What I was actually wondering was whether we need a `--dashed-ident` feature somewhere. This is probably better.

Reply:
Okay, yeah maybe we need a `--dashed-ident` feature somewhere but I think for now I will land this info here.